modules/ROOT/pages/websockets/websocket-about-websocket-connector.adoc

```diff
@@ -0,0 +1,60 @@
+= About the Websockets Connector (Design Center)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== Why use Websockets
+
+WebSockets are a key piece of modern integration due to their ability to:
+
+* Seamlessly work on top of the existing HTTP infrastructure.
+* Avoid polling
+* Implement server push patterns
+* Establish bi-directional and full duplex communication between server and client.
+
+== Basic components
+
+There are two basic socket types:
+
+* Inbound: represents a websocket that will open upon receiving an incoming connection request from a client that supports websocket communication.
```

Comment:
Clarify that inbound sockets are initiated by external systems that connect through a WebSocket endpoint that the Mule application exposes and is listening on